erpm_gain: 3500.0 
steering_max: 0.488
wheelbase: 0.29
cg_height: 0.136
wheelspeed_max: 17.0
nominal_voltage: 14.8
motor_kv: 3930
speed_control_kp: 1.0
speed_control_ki: 1.0
safe_mode: true
track_width: 0.25
accel_gain: 1.0
roll_gain: 0.33
steer_slack: 0.0
LPF_tau: 0.2